project/CopyDocs.scala

```diff
@@ -19,54 +19,44 @@ object CopyDocs {
    */
   val outputDir = FileSystems.getDefault.getPath("docs-for-dotty-page")
```

Comment:
Why do we still need this file? I thought the idea would be to not copy documentations anymore.

Reply:
Yes, actually there are some more things to be cleaned, therefore I think it would be better to do it in a follow-up PR. For now, I scoped to add features without breaking current base logic